pages/plot-data-from-csv.py

```diff
@@ -3,21 +3,22 @@
 from utils.code_and_show import example_app
 
 
-dash.register_page(__name__, description="Plotting data from csv file")
+dash.register_page(__name__, description="This application plots stocks data collected from csv. It contains button to invert axis on graph.")
```

Comment:
The second sentence indicates the layout type and the callback type, spelled exactly as in the [Apps Progress Sheet](https://docs.google.com/spreadsheets/d/1-ezhib_RuwSDckDM2znEkIutO7MEOltQYqkc_61zUcU/edit?usp=sharing). You can find an example [in this issue](https://github.com/AnnMarieW/dash-app-gallery/issues/51#issue-1273837546).
Please update the second sentence to the following:

"It has a top-bottom layout and a regular-callback."